src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -1674,7 +1674,7 @@ internal long SetRequestContent(HttpRequestMessage request, string content)
                 // would be used if Charset is not supplied in the Content-Type property.
                 try
                 {
-                    var mediaTypeHeaderValue = new MediaTypeHeaderValue(ContentType);
+                    var mediaTypeHeaderValue = MediaTypeHeaderValue.Parse(ContentType);
```

Comment:
We could use MediaTypeHeaderValue.TryParse()

Reply:
No need. we already have logic in place to handle the exceptions and perform the correct operations.

Reply:
We should _avoid_ exceptions if we can. If the exceptions come only from the Parse() method we should use TryParse().

Reply:
Except that we have different behavior depending on the type of exception. We would hand to build our own logic to figure that out. When the type of exception determines behavior it is perfectly acceptable to use them in a try/catch.

Reply:
My comment is not blocking.